### PR TITLE
Fine-tune the plugin description

### PR DESCRIPTION
**Pull request by smcvb**
This pull request adjusts the plugin description slightly, by:

- Adding DDD to supported concepts of the framework
- Adding Event Sourcing to supported concepts of the framework
- Adding message-driven architecture to supported concepts at the core of providing DDD, CQRS, and ES
- Specify the plugin only supports version 4.x and higher
- Make minor adjustments according to Grammarly

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -22,9 +22,10 @@
 
     <description>
         <![CDATA[
-        Supports developers while writing applications using <a href="https://axoniq.io/">Axon Framework</a>. <br/><br/>
-        Axon Framework helps build scalable, extensible and maintainable applications by supporting developers apply the Command Query
-        Responsibility Separation (CQRS) architectural pattern.<br/><br/>
+        Supports developers while writing applications using <a href="https://axoniq.io/">Axon Framework</a> version 4.x.<br/><br/>
```
Comment (lfgcampos) [inline src/main/resources/META-INF/plugin.xml:25]:
```suggestion
        Supports developers while writing applications using <a href="https://axoniq.io/product-overview/axon-framework">Axon Framework</a> version 4.x.<br/><br/>
```

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -22,9 +22,10 @@
...
+
+        Axon Framework helps build scalable, extensible, and maintainable applications by supporting developers to apply Domain-Driven Design (DDD), Command-Query Responsibility Separation (CQRS),
+         and Event Sourcing. The framework provides a message-driven architecture with dedicated command, event, and query messages, to support these paradigms.<br/><br/>
```
Comment (lfgcampos) [inline src/main/resources/META-INF/plugin.xml:28]:
```suggestion
         and Event Sourcing (ES). The framework provides a message-driven architecture with dedicated command, event, and query messages, to support these paradigms.<br/><br/>
```

just to follow the others =)

**Review by lfgcampos — APPROVED** [review]

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (smcvb) [reply src/main/resources/META-INF/plugin.xml]: good catch!

File: src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (smcvb) [reply src/main/resources/META-INF/plugin.xml]: good point!

**Review by Morlack — COMMENTED** [review]
Looked good, merged!

**Pull request merged**